### ISSUE TITLE
Ordering siblings in descending order causes problems with retrieving ancestors

### ISSUE THREAD

**Issue by benjaminLevinson** [feincms/django-tree-queries#60]
Ordering siblings in descending order and then calling "ancestors" is raising an error.

Here is my model:
```
class Post(tree_queries.models.TreeNode):
    created = models.DateTimeField(auto_now_add=True)
    parent = tree_queries.models.TreeNodeForeignKey(
        "self",
        blank=True,
        null=True,
        on_delete=models.CASCADE,
        verbose_name="parent",
        related_name="children",
    )

    class Meta:
        ordering = ["-created"]
```

In a template, I call:
```
    {% for ancestor in post.ancestors.all %}
    ( do stuff
    {% endfor %}
```

Result:
```
ProgrammingError
column "-created" does not exist
LINE 11:             array["-created"]::text[] AS tree_ordering,
                           ^
HINT:  Perhaps you meant to reference the column "t.created".
```

Am I doing something wrong? It seems like there must be an issue with how `ancestors` deals with sibling ordering.

**Comment by matthiask** [comment]
Hi @benjaminLevinson 

Unfortunately descending ordering isn't supported yet, see https://github.com/feincms/django-tree-queries/issues/6

**Comment by matthiask** [comment]
Fixed by #62, I think.
